src/components/application_manager/src/resumption/resumption_data_db.cc

```diff
@@ -401,32 +406,6 @@ void ResumptionDataDB::GetDataForLoadResumeData(
   SelectDataForLoadResumeData(saved_data);
```

Comment:
@Kozoriz its just that the function return with resuly on one place and with false on the other.. I was thinking if the declaration is made before ExistenceCheck.. and if we fail to return result ..